src/modules/translation/languages/es.json

```diff
@@ -196,6 +196,7 @@
   },
   "create_single_item_modal": {
     "title": "Nuevo item",
+    "thumbnail_step_title": "Editar el thumbnail",
```

Comment:
I used `thumbnail` because is not translated in our spanish keys at the moment:
![image](https://user-images.githubusercontent.com/8763687/180718057-2dd92176-edb0-4112-9e9f-1861df8f400a.png)

![image](https://user-images.githubusercontent.com/8763687/180718079-f05a4bde-0240-45c5-b5bd-07db81e999ae.png)